### ISSUE TITLE
Rampage exponential input inverted

### ISSUE THREAD

**Issue by synthi** [VCVRack/Befaco#1]
I don´t have the rampage but the serge DUSG. theCV inputs in rampage are inverted so sending a 1v/oct signal to the exp input it goes lower in frequency as you go up in the keyboard. Using an inverter in between as a workaround but I think the CV inputs must be reversed..

**Comment by AndrewBelt** [comment]
Can confirm that Exp CV is inverted from Rise/Fall CV on Rampage, and doesn't track 1V/oct. However, VCV Rampage Exp CV seems to be about 10 times more aggressive than Befaco Rampage, so I'll change that in the next release.

Here's what someone said about the tracking https://www.muffwiggler.com/forum/viewtopic.php?p=1936120#1936120

**Comment by synthi** [comment]
I see.. Strange since is more or less ac serge clone with some adds.
So the rise/fall cv inputs are inverted, right. but the exp Cv in must not be inverted right? Also please left it "calibrated" for v/oct! I see is actually calibrated for 1v/oct they just relabeled the 1v/oct to exp input because the poor tracking (1-2 octaves) but its still 1v/oct

**Comment by AndrewBelt** [comment]
Even though Rampage isn't calibrated for 1V/oct (it is impossible without hardware modifications), I *could* make it track in software. However, I had a choice to design Rampage well for audio rate frequencies, or make it low CPU, and I chose the latter. So there are additional problems with using VCV Rampage as an oscillator if that's what you're doing.

**Comment by synthi** [comment]
But it vtrackng right now..!! Is just reversed... I add a befaco attnuverter set at -1 and it tracks several octaves, just have a limit in the high register at about 600hz ( could other freq I'm telling from ear and from memory)

**Comment by AndrewBelt** [comment]
I don't know why you're getting different results as I am, but when I track VCV Rampage with the Exp CV in, I have to set the attenuverter to about 1/10, not -1.

**Comment by synthi** [comment]
Yes Andrew, you are right! I was speaking from memory and it seem that is not so good as used to be... I tried several combos for getting 1v/oct and what Im using is 1v/pct to attenunverter at -1 and then to both rise and fall cv inputs. I couldnt get tracking from the exp input

**Comment by synthi** [comment]
OK! that make sense, in the hardware the exp input should be just a inverting amp without gain normalized to both rise and fall cv inputs. Is great to have raage, I'm usoed to the DUSGs in my serge system, I have 6 of them(!) and usually I make entire patches just with those 😀

**Comment by AndrewBelt** [comment]
What maximum frequency is the DUSG capable of, without patching hacks? I've been considering a commercial Serge port after asking Serge about licensing.

**Comment by synthi** [comment]
About 4 to 5Khz, but what do you mean by "without patching tricks"? you must patch the gate out to trigger input for oscillation.... Would be awesome a serge collection with the most important modules. IMO those are the DUSG, TWS, WCM, SSG, NCOM, VCFQ, Programmer/sequencer and WAD

**Comment by AndrewBelt** [comment]
Meaning without sending a DC voltage to Rise or Fall or 1V/oct to make it go higher than what the knobs are capable of doing.
Yup, all of those are planned. I'm only really interested if I can get very Serge-like panels, so I'd need Serge's okay for that.
